integration-test/dns-api-mesos/src/main/resources/application.conf

```diff
@@ -37,7 +37,8 @@ akka {
       contact-point-discovery {
 
         # Define this name to be looked up in service discovery for "neighboring" nodes
-        # If undefined, the name will be extracted from the ActorSystem name
+        # If undefined, the name will be taken from the AKKA_CLUSTER_BOOTSTRAP_SERVICE_NAME
+        # environment variable or extracted from the ActorSystem name
```

Comment:
The comment doesn't seem to match the definition.

Reply:
How do you mean? If `service-name` is undefined in this `application.conf` it will fall back to the `reference.conf` above, where `service-name` is taken from the `AKKA_CLUSTER_BOOTSTRAP_SERVICE_NAME` environment variable if filled, or otherwise falls back to `<service-name>` which gets replaced by the actor system name?